service/key/key.go

```diff
@@ -20,18 +20,10 @@ func ClusterCustomer(customObject kvmtpr.CustomObject) string {
 	return customObject.Spec.Cluster.Customer.ID
 }
 
-func ClusterDomain(sub, clusterID, domain string) string {
-	return fmt.Sprintf("%s.%s.%s", sub, clusterID, domain)
-}
-
```

Comment:
These functions are not used at all so I removed them.

Reply:
I am a bit confused, why do these functions exist if they aren't used? Dead weight from previous changes?

Reply:
I think somebody forget to remove them when the used code got changed. I don't know.